vendor-extra/ViewsBundle/src/Views/ViewConverterRegistry.php

```diff
@@ -5,10 +5,11 @@
 namespace Libero\ViewsBundle\Views;
 
 use FluentDOM\DOM\Element;
-use FluentDOM\DOM\Node\NonDocumentTypeChildNode;
+use FluentDOM\DOM\Node;
 use LogicException;
 use function array_push;
 use function is_string;
+use function var_dump;
```

Comment:
added, but unused; I independently saw signs that this class need to be made more prominent/documented/observable as it's a junction for many different views and nodes; logging could help

Reply:
Switching to using the Symfony event dispatcher would provide a bunch of stuff out of the box (logging, profiler pane) rather than having to create it. Also need to setup Xdebug.

Reply:
#57